algorithms/max-flow/max-flow.tex

```diff
@@ -0,0 +1,82 @@
+\section{Max Flows}\index{graph!Max Flows}
+#ifdef hackpackpp
+There are several types of problems that involve finding maximum flow in a graph.
+Many of these problems can easily tackled by making small transformations to the graph.
+
+
+If there is more than one source or sink, one can add a "super" source and or sink that connects to all of the sources and sinks.
+Then find the flow from the super source to the super sink.
+
+
+If the graph is undirected, simply insert every edge twice in both directions.
```

Comment:
Should this really be the case if the graph is _undirected_, or should it be if the graph is _directed_?